includes/admin.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * Insecure Content Warning assets
+ *
+ * @package ICW
+ */
+
+namespace ICW\Admin;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Setup actions and filters
+ */
+function setup() {
+	add_action( 'admin_menu', __NAMESPACE__ . '\\admin_menu' );
+}
+
+/**
+ * Register the admin menu.
+ *
+ * @return void
+ */
+function admin_menu() {
+	$hook = add_management_page(
+		__( 'Insecure Content Warning Admin', 'insecure-content-warning' ),
+		__( 'Insecure Content Warning', 'insecure-content-warning' ),
+		'install_plugins',
```

Comment:
```suggestion
		'edit_posts',
```

As a general capability, this seems closes to what the page is doing.

Reply:
Good point, thank you!